torch/_inductor/codecache.py

```diff
@@ -612,7 +625,8 @@ class InvalidVecISA(VecISA):
     def __str__(self) -> str:
         return "INVALID_VEC_ISA"
 
-    def __bool__(self):
+    @functools.lru_cache(None)
```

Comment:
This is bug prone behavior to LRU cache on methods (attempts to cache self which is often bad). I really need to enable that flake8-bugprone rule.